pyproject.toml

```diff
@@ -19,19 +30,22 @@ extend-exclude = '''
 )/
 '''
 
-[tool.pytest.ini_options]
-minversion = "6.0"
-addopts = "-ra --doctest-glob='*.rst'"
-testpaths = [
-"docs/conversion_examples_gallery/",
-"tests"
+
+
+[tool.ruff]
+exclude = [
+  "*/__init__.py"
```

Comment:
how comes we exclude these ones?

Reply:
Beacuse we don't fully use sklearn convention with exposed imports and that flags unfixable F401's since the things we import in those files aren't in `__all__` for that submodule